tests/flows/validation/test_base.py

```
@@ -135,61 +79,40 @@ def test_add_default_lines_success(order_factory, mocker):
+    master_payer_account_id_param = get_ordering_parameter(
+        OrderParametersEnum.MASTER_PAYER_ACCOUNT_ID.value, result
+    )
+    assert new_account_instructions_param["constraints"]["hidden"] is True
+    assert new_account_instructions_param["constraints"]["required"] is False
+    assert master_payer_account_id_param["constraints"]["hidden"] is False
+    assert result["lines"] == [{"item": mock_items[0], "quantity": 1}]
 
 
-def test_validate_order_orchestrates_all_steps(order_factory, order_parameters_factory, mocker):
+def test_validate_order_returns_error_when_new_account_instructions_visible(
+    order_factory, order_parameters_factory, mocker
```

Ruff (0.14.14)

[warning] Unused function argument: `mocker`

(ARG001)